layouts/partials/hooks/head-end.html

```diff
@@ -13,3 +13,13 @@
 {{- if .HasShortcode "cncf-landscape" -}}
 <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js" integrity="sha384-hHTwgxzjpO1G1NI0wMHWQYUxnGtpWyDjVSZrFnDrlWa5OL+DFY57qnDWw/5WSJOl" crossorigin="anonymous"></script>
 {{- end -}}
+
+{{ if .HasShortcode "mermaid" }}
+  <!-- Copied from https://unpkg.com/mermaid@10.6.1/dist/mermaid.min.js -->
```

Comment:
I don't see how this comment makes sense anymore It might at some point in the past when part of the .min.js script was embedded here, but that clearly is no longer the case.

```suggestion
```

Reply:
Why does this no longer make sense? The point is to let people confirm the value of `integrity` and also to understand / verify the provenance.

Reply:
Yep, the comment is in the place where we do the Get so that the file contents 100% match the minified JS. We prefer to have the files be 100% the same.

One day we might be able to load Mermaid from an external CDN.